core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/saved-workflow-section.component.ts

```diff
@@ -522,8 +521,12 @@ export class SavedWorkflowSectionComponent implements OnInit, OnChanges {
 
     if (this.selectedProjects.length !== 0) {
       searchOutput = searchOutput.filter(workflowEntry => {
-        this.selectedProjects.some(project => workflowEntry.projectIDs.includes(project.pid));
-      });
+        for (const proj of this.selectedProjects) {
+          if (workflowEntry.projectIDs.includes(proj.pid)) {
```

Comment:
I brought back the old for loop, because the new version with `some()` would always return an empty array.